### ISSUE TITLE
Include offline test

### ISSUE THREAD

**Issue by ryanmoran** [paketo-buildpacks/dotnet-publish#77]
This buildpack should include a test that confirms that offline vendoring of packages can be achieved.

You can vendor offline dependencies into your app by running the following command from within your source directory:

```
dotnet restore --packages ./packages --runtime ubuntu.18.04-x64
```

This will download all the packages for your app into the `./packages` directory. You then need to configure NuGet to look only in this location to find packages. Create a `nuget.config` file with the following contents:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
 <packageSources>
    <clear />
    <add key="app" value="./packages" />
 </packageSources>
</configuration>
```

You can then run a `pack build` with the `--network` flag set to `none` to confirm that the buildpack supports offline vendoring of packages.

**Comment by fg-j** [comment]
@thitch97 's contribution guide lives here: https://github.com/paketo-buildpacks/.github/blob/main/CONTRIBUTING.md

**Comment by ForestEckhardt** [comment]
This issue was addressed by PR #138.
